plugin-build

```diff
@@ -111,7 +111,9 @@ if ( file_exists( $pre_zip_hook ) ) {
 	include $pre_zip_hook;
 }
 
-$zip_name = "$plugin_slug-$version.zip";
+if ( ! isset( $zip_name ) ) {
+	$zip_name = "$plugin_slug-$version.zip";
```

Comment:
We need to be able to pass the equivalent of the `$plugin_slug` part of the filename, as currently https://github.com/deliciousbrains/wp-aws/pull/381/files#diff-d5457f459177e063f0580f2b029ccc86R6 means that the zip files get created without the version number in the filename